src/moduleLoader.js

```diff
@@ -40,19 +44,36 @@ const moduleLoader = function(options = {}) {
     }
 
     function loadModules(els) {
+        const loadEsm = settings.loadingMethod === 'esm' || (settings.loadingMethod === 'auto' && dynamicImportsSupported === true);
         els.forEach(el => {
             const module = el.getAttribute(settings.moduleDataAttr);
+            const resolvedBasepath = el.dataset.basepath || settings.basePath;
+            const resolvedSuffix = el.dataset.suffix || settings.filenameSuffix;
+            const modulePath = `${resolvedBasepath}${module}${resolvedSuffix}`
             const props = {
                 containerElement: el
             };
-            System.import(module).then(mod => mod.default(props));
+            if (loadEsm) {
+                import(modulePath).then(mod => mod.default(props));
+            } else {
```

Comment:
If the user specifies the loadingMethod to be 'esm', then moduleLoader should never fall back to SystemJS, right?

Reply:
NO!